modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/KotlinSpringServerCodegen.java

```diff
@@ -440,7 +451,7 @@ public void processOpts() {
 
     @Override
     public void addOperationToGroup(String tag, String resourcePath, Operation operation, CodegenOperation co, Map<String, List<CodegenOperation>> operations) {
-        if (library.equals(SPRING_BOOT) && this.delegatePattern) {
+        if (library.equals(SPRING_BOOT) && !useTags) {
```

Comment:
This changes previous behavior for `library.equals(SPRING_BOOT) && this.delegatePattern` and would be a breaking change. Was this intentional? If so, can you speak toward the intention? To me it looks like `this.delegatePattern` was accidentally removed.

Reply:
Thanks for the comment Jim. The replacement of this.delegatePattern with useTags was intentional. I'm currently using the delegate pattern with tags in my Java project and this change was needed to get the Kotlin generator to produce equivalent code to the Java one. In fact if you look at SpringCodegen.java you can see it does something very similar
{code}
if ((library.equals(SPRING_BOOT) || library.equals(SPRING_MVC_LIBRARY)) && !useTags) {
{code}
I understand that this is a breaking change but I'm not sure we can support tags properly while keeping backwards compatibility.

Reply:
Fair enough. I'll mark this PR as a breaking change. Thanks for the explanation.